compiler/rustc_error_codes/src/error_codes/E0574.md

```diff
@@ -4,9 +4,9 @@ expected.
 Erroneous code example:
 
 ```compile_fail,E0574
-mod Mordor {}
+mod mordor {}
 
-let sauron = Mordor { x: () }; // error!
+let sauron = mordor { x: () }; // error!
```

Comment:
It's the usage *of a module* in struct position that's problematic, so I'd argue the module is correctly snake_case... but I guess it's just a matter of perspective: did they mean to write `struct Mordor {}` or did they mean to write `let sauron = Sauron { x: () };` (or similar)?

Reply:
If someone tried to use a module as a struct, it's possible they made a naming convention error too. (Maybe they used snake_case for the struct instead of UpperCamelCase for the module). It's not clear to me, however, that violating naming conventions in the error description is helpful.